docs/docs/07-forms.md

```diff
@@ -50,7 +50,7 @@ This will render an input that always has a value of `Hello!`. Any user input wi
   },
   render: function() {
     var value = this.state.value;
```

Comment:
`value` is defined here, so it's perfectly valid.  This is most likely done for formatting reasons here.